app/scripts/views/settings.js

```diff
@@ -148,6 +149,32 @@ function ($, modal, Cocktail, Session, BaseView, AvatarMixin,
       return self._showAvatar();
     },
 
+    beforeDestroy: function () {
```

Comment:
No need for this, it's done [automatically over in base.js](https://github.com/mozilla/fxa-content-server/blob/16c9fd54962e44292ec94a4305399bcacfa2d3cb/app/scripts/views/base.js#L373-L375)